libevm/tooling/release/cherrypick.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
since this is adding some level of bash scripting, perhaps consider taking the `shellcheck.sh` and calling it in CI.

Reply:
Adding `.github/workflows/shellcheck.yml` as it's a simpler way to invoke it.

Reply:
I enabled the workflow before doing the fixes to [demonstrate that it works](https://github.com/ava-labs/libevm/actions/runs/13332830033/job/37240935514).

Thanks for the suggestion!